src/box2dLight/SmoothChainLight.java

```diff
@@ -0,0 +1,275 @@
+package box2dLight;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+import com.badlogic.gdx.math.MathUtils;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.physics.box2d.Body;
+import com.badlogic.gdx.utils.Array;
+
+/**
+ * This is a composite of multiple line and cone lights to
+ * Created by PiotrJ on 16/11/2015.
+ */
+public class SmoothChainLight extends Light implements DebugLight {
+	private int side = 1;
```

Comment:
Which side the light will appear on. If you have a one chain link, a line, 1 will make the light go up, -1 will go down. Or other way around. Its same as default chain light